frontend/src/toolbar/button/ToolbarButton.scss

```diff
@@ -89,3 +92,7 @@
     border-radius: 4px;
     font-weight: bold;
 }
+
+.Popup {
+    z-index: 2147483020;
+}
```

Comment:
This global high level override of a totally unconnected css class definitely should not belong in `ToolbarButton.scss` :).

Why not put it next to all the other global toolbar style overrides... in styles.scss?